speed-dreams/source-2.2.3/cmake/thirdpartydeps.cmake

```diff
@@ -559,6 +559,22 @@ MACRO(ADD_IPCLIB_PACKAGE TARGET)
 
 ENDMACRO(ADD_IPCLIB_PACKAGE TARGET)
 
+MACRO(ADD_MYSQL_PACKAGE TARGET)
+
+  IF(MYSQL_FOUND)
+    INCLUDE_DIRECTORIES(${MYSQL_INCLUDE_DIR})
+    TARGET_LINK_LIBRARIES(${TARGET}
+            ${MYSQL_LIBCRYPTO_LIBRARY}
+            ${MYSQL_LIBSSL_LIBRARY}
+            ${MYSQL_CPPCONN_LIBRARY}
+            ${MYSQL_CPPCONN8_LIBRARY}
+            ${MYSQL_CPPCONN_STATIC_LIBRARY}
```

Comment:
Adding static libraries without dll will crash the build if in the future we accidentally use the static functions. Either add the static library dlls to customthirdparty.cmake or remove them here and remove them from checks.cmake if they are not used

Reply:
Commented out the static ones. If you're okay with that please resolve the conversations regarding this issue